docs/source/relationships.rst

```diff
@@ -0,0 +1,235 @@
+Relationships
+=============
+
+
+Basics
+------
+
+Relationships in Ramses are used to represent One-To-Many(o2m) and One-To-One(o2o) relationships between objects in database.
+
+To set up relationships fields of types ``foreign_key`` and ``relationship`` are used. ``foreign_key`` field is not required when using ``nefertari_mongodb`` engine and is ignored.
+
+
+For this tutorial we are going to use example of users and stories. In this example we have OneToMany relationships betweed User and Story: One user may have many stories but story has only one owner.
+Check the end of the tutorial for a complete example of RAML file and schemas.
+
+Example code is the very minimum needed to explain the subject. We will be referring to the examples along all the tutorial.
+
+
+relationship
+------------
+
+Must be defined on *One* side of OneToOne or OneToMany relationship (``User`` in our example). Relationships are created as OneToMany by default. To create OneToOne relationships, specify ``"uselist": false`` in ``_db_settings`` of this field.
+
+Example of using ``relationship`` field:
+
+.. code-block:: json
+
```

Comment:
You should probably include `class User...` or something like that at the top of the snippet to reinforce that this field is on the `User` model.